test/SerialIOTest.cpp

```diff
@@ -725,7 +724,7 @@ void fileBased_write_test(const std::string & backend)
         o.setOpenPMDextension(1);
         o.iterations[3].setTime(static_cast< double >(3));
         o.iterations[4].setTime(static_cast< double >(4));
-        o.flush();
+        //o.flush();
```

Comment:
removing that is confusing because it is a late flush. this should be very similar to the destructor which is called next and flushes, too.

what I thought we chatted about was to set for ADIOS the `setOpenPMDextension` before the first iteration's flush occurs?

Reply:
Oh I thought I explained that the attributes in adios1  are written when the series is deleted, so it is not really an overwritten issue. 
I can put this flush back. Thought it is not needed. Is there a reason to have it there?

Reply:
OK I looked in the history flush was there. I put it back. Sorry

Reply:
Nothing to be sorry about, thanks a lot :)